### PR TITLE
Update Basic Tutorial Episodes for compatibility with v2.10.0

### PR DESCRIPTION
**Pull request by rswamina**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes [316](https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/316)

### CODE REVIEW

**Comment by rswamina** [comment]
@LisaBock   - Please review when possible. Thanks!

File: _episodes/04-recipe.md [hunk 1]
```diff
@@ -368,7 +368,7 @@ Do you recognize the basic recipe structure that was introduced in episode 1?
 > >   which contains preprocessed data for each of the input datasets, plus a
 > >   combined `MultiModelMean`, a `metadata.yml` file and provenance files.
 > > - **map/script1**: creates `/run/map/script1` with general information and a
-> >   log of the diagnostic script run. It also creates `/plots/map/script1` and
+> >   log of the diagnostic script run. It also creates `/plots/map/script1/png` and
```
Comment (LisaBock) [inline _episodes/04-recipe.md:371]: If you want to add the "png" subfolder, you may do it also for the timeseries plots folder below.

**Review by LisaBock — APPROVED** [review]
Thanks Ranjini! Depart a small comment everything seems fine. 

Fortunately there were not many changes needed.

File: _episodes/04-recipe.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rswamina) [reply _episodes/04-recipe.md]: Good point. I think I will leave this out. That way it is consistent for png and other file types too. Thanks!

**Comment by rswamina** [comment]
Thanks @LisaBock for the quick review.
All checks have passed and reviewer comments addressed so changes will be merged.

**Pull request merged**